src/test/java/tests/StakeKeyTests.java

```diff
@@ -31,4 +33,45 @@ public Object[][] DatasetListAddress(){
                 {1234}
         };
     }
+    @Test(description = "get stake instantaneous rewards with stake key", groups = {"stakeKey"}, dataProvider = "stakeKeyInstantaneousReward")
+    public void getStakeInstantaneousReward(Object stakeKey){
+        stakeKeySteps.getStakeInstantaneousRewards(stakeKey)
+                .validateStatusCode(HttpURLConnection.HTTP_OK);
+    }
+    @DataProvider(name = "stakeKeyInstantaneousReward")
+    public Object[][] DatasetStakeKeyInstantaneousRewards(){
+        return new Object[][]{
+                {"stake_test1urz3dwcwdhvl9dy75rj5krvpmjyldeyxqx5p6xms6xyjd7quqmq2p"},
```

Comment:
should put on top

Reply:
this stakeKey is a invalid, so i think we shouldn't put on top